src/api/handlers/guild.ts

```diff
@@ -731,8 +731,13 @@ export function leaveGuild(guildID: string) {
   return RequestManager.delete(endpoints.GUILD_LEAVE(guildID));
 }
 
+/** Returns an array of voice regions that can be used when creating servers. */
+export function getVoiceRegions() {
+  return RequestManager.get(endpoints.VOICE_REGIONS);
+}
+
 /** Returns a list of voice region objects for the guild. Unlike the similar /voice route, this returns VIP servers when the guild is VIP-enabled. */
-export function getVoiceRegions(guildID: string) {
+export function getGuildVoiceRegions(guildID: string) {
```

Comment:
Separate PR for this.

Reply:
I didn't do a separate PR because the function added actually has the same name.

Reply:
I don't think this is resolved, this makes this pr breaking. Rename the other one to something different like `getAllVoiceRegions` and in a separate pr for.

OOOOR

keep the 1 function but make guildID optional.

if (!guildID) get them all

Reply:
if the guild is a vip you get all. `getVoiceRegions` only returns available ones so no vip voice servers.

Reply:
So you should create a separate PR changing the name of this function.

Reply:
How? The added function literally has the same name. If you have a better name I'll change it.

Reply:
getAllAvailableVoiceRegions